xarray/core/formatting.py

```diff
@@ -334,7 +334,10 @@ def summarize_variable(
     else:
         dims_str = ""
 
-    nbytes_str = f" {render_human_readable_nbytes(variable.nbytes)}"
+    try:
+        nbytes_str = f" {render_human_readable_nbytes(variable.nbytes)}"
+    except TypeError:
+        nbytes_str = " ?"
```

Comment:
This is a bit ugly, but we'd need to define `nbytes` on pandas wrappers, and it breaks every repr in our tests :/